<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationButtonGroup</class>
 <widget class="QWidget" name="OperationButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>36</height>
   </rect>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>35</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="next10Button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>10 moves later  [Alt + ↓]</string>
   </property>
   <property name="text">
    <string>N10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevCandidateButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Previous candidate  [←]</string>
   </property>
   <property name="text">
    <string>PC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Previous  [↑]</string>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Next  [↓]</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextCandidateButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Next candidate  [→]</string>
   </property>
   <property name="text">
    <string>NC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prev10Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>10 moves earlier  [Alt + ↑]</string>
   </property>
   <property name="text">
    <string>P10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lastButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>Last position  [Ctrl + ↓]</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="firstButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>2</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>First position  [Ctrl + ↑]</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
